src/main/java/hellos/Hellos.java

```diff
@@ -33,4 +34,8 @@ private String kkSaysHello() {
     return "KK says 'Hello!'\n";
   }
 
+  private String elenaSaysHello() {
+    return "Elena says 'Xin chào!'\n";
```

Comment:
I think you make some valid points here. Nice argument.

Reply:
I still think it's cool that we have a little Vietnamese in the project. 😎